rest_framework/filters.py

```diff
@@ -85,7 +89,8 @@ def get_search_terms(self, request):
         """
         value = request.query_params.get(self.search_param, '')
         field = CharField(trim_whitespace=False, allow_blank=True)
-        return field.run_validation(value)
+        cleaned_value = field.run_validation(value)
+        return search_smart_split(cleaned_value)
```

Comment:
This could have been done by simply changing to 

```python
return list(search_smart_split(cleaned_value))
```

There is no point to changing the return value and definition of `search_smart_split`.